src/picasso-definition/components/navigation-panel/index.js

```diff
@@ -3,9 +3,10 @@ import KEYS from '../../../constants/keys';
 import translate from '../../../utils/math/translate';
 import zoom from '../../../utils/math/zoom';
 
-export default function createNavigationPanel({ layoutService, chartModel }) {
+export default function createNavigationPanel({ layoutService, chartModel, context }) {
```

Comment:
Since it's only rtl you are using in here, you might as well just pass `rtl` in here...

Reply:
fixed!